packages/backend/src/apps/lettersg/auth/verify-api-key.ts

```diff
@@ -0,0 +1,16 @@
+import { IGlobalVariable } from '@plumber/types'
+
+export async function verifyApiKey($: IGlobalVariable): Promise<void> {
+  try {
+    await $.http.get('/v1/templates?limit=1')
+  } catch (err) {
+    if (err.response.status === 401) {
+      throw new Error(
+        'API key is invalid, please ensure you have copied the correct API key',
+      )
+    }
+    throw new Error(
+      'Unknown error, please contact support@plumber.gov.sg for more assistance.',
+    )
```

Comment:
would it be a good idea to put in additional logging here for further investigation? if we throw an error with just this error message, I think we will lose the HTTP status and any informative error messages from LetterSG.

Reply:
ok thks for pointing out, i realise that I should throw back the `err` here and log it under the graphql query instead